build.assets/Dockerfile

```diff
@@ -147,7 +147,7 @@ RUN (curl -L https://github.com/golangci/golangci-lint/releases/download/v1.46.0
      rm -r golangci-lint*)
 
 # Install GCI
-RUN go install github.com/daixiang0/gci@v0.8.2
+RUN go install github.com/daixiang0/gci@v0.7.0
```

Comment:
Out of curiosity, why the downgrade?

Reply:
GCI also runs goimports and 0.8.0+ also format comments in the same way as go 1.19 and  golangci-lint complaints https://github.com/daixiang0/gci/pull/111/files#diff-74342d74a10a1ff588f6ae1b4e02e5869a759bcdad74b2f41756252ef024b72eR70